custom_components/localtuya/cover.py

```diff
@@ -43,7 +43,7 @@
 
 DEFAULT_NAME = 'localtuyacover'
 
-REQUIREMENTS = ['pytuya>=7.1.0']
```

Comment:
You can remove references to `REQUIREMENTS` everywhere, they haven't been used since manifests were introduced (and this wouldn't work either as the version would need to exist on pypi).

Reply:
OK, done